docker-compose.yml

```diff
@@ -33,7 +35,7 @@ services:
 
   db:
     networks: [pah]
-    image: postgres:9.5
+    image: postgres:14.4
```

Comment:
just a note - this will cause the code to be misaligned with real DB used on prod & dev environments, so it's potentially breaking change. _Likely_ nothing should change, because we're using only basic capabilities of PostgreSQL and also ORM should cover any differences, but still, it's an important one to note.